core/src/main/java/org/mule/runtime/core/internal/serialization/AbstractSerializationProtocol.java

```diff
@@ -19,12 +21,15 @@
 import java.io.InputStream;
 import java.io.OutputStream;
 
+import javax.inject.Inject;
+
 /**
- * Base class for implementations of {@link SerializationProtocol} This class implements all the base behavioral
- * contract allowing its extensions to only care about the actual serialization/deserialization part.
+ * Base class for implementations of {@link SerializationProtocol} This class implements all the base behavioral contract allowing
+ * its extensions to only care about the actual serialization/deserialization part.
  */
 public abstract class AbstractSerializationProtocol implements SerializationProtocol, MuleContextAware {
```

Comment:
Do we still need MuleContextAware class?

Reply:
no, but the setter is still needed becuause of how the lifecycle of these objects is handled